src/modules/MeasureTool/MeasureToolUI/MeasureToolXAML/MainWindow.xaml.cs

```diff
@@ -56,6 +56,11 @@ public MainWindow(PowerToys.MeasureToolCore.Core core)
             this.SetIsMaximizable(false);
             IsTitleBarVisible = false;
 
+            // Remove the caption style from the window style. Windows App SDK 1.6 upgraded added it, which made the title bar and borders appear for Measure Tool. This code removes it.
```

Comment:
is this a bug on their side?

Reply:
This could be due to a change w/ 1.6 and the fact we use WinUIEx for that